src/test/java/com/google/javascript/gents/singleTests/module_class3.js

```diff
@@ -5,7 +5,7 @@ class Klass {
 
   /** 
    * @param {number} n
-   * @param {Array<?>=} list
```

Comment:
Why delete the '='? I looked around and it appears closure allows one to write this with or without the '=', but the one with '=' is more common.

Reply:
Reverted.